src/shared/services/document-management/document-management.service.ts

```diff
@@ -22,8 +23,11 @@ export class DocumentManagementService {
     headers.append(DocumentManagementService.HEADER_CONTENT_TYPE, null);
     return this.http
       .post(url, formData, { headers })
+      .pipe(delay( 3000 ))
```

Comment:
move 3000 to a constant and add an explanatory comment on why this has been added please